app/src/main/javascript/bundles/experiment-page/src/TSnePlotViewRoute.js

```diff
@@ -308,7 +310,12 @@ TSnePlotViewRoute.propTypes = {
     value: PropTypes.string.isRequired
   }).isRequired).isRequired,
   selectedK: PropTypes.number,
-  anatomogram: PropTypes.object.isRequired
+  anatomogram: PropTypes.object.isRequired,
+  initialCellTypeValues: PropTypes.array
```

Comment:
Array of what? I think you can be a bit more specific.

Reply:
Makes sense will update!